blocks/form/form.js

```diff
@@ -0,0 +1,267 @@
+const formatFns = await (async function imports() {
```

Comment:
Though await is supported as top level imports but not sure what its impact will be on performance